model/src/pyrenew/model/rtinfectionsrenewalmodel.py

```diff
@@ -299,48 +299,64 @@ def sample(
         # Sampling from Rt (possibly with a given Rt, depending on
         # the Rt_process (RandomVariable) object.)
         Rt, *_ = self.sample_rt(
-            duration=n_timepoints,
+            n_timepoints=n_timepoints,
```

Comment:
This naming (i.e. `n_timepoints`) might change with or guide further discussion regarding the representation of time in this codebase. It is difficult for me to see what is better than `n_timepoints` (maybe `n_timeunits`?) but I thought this was worth flagging.